src/server/src/main/java/io/cassandrareaper/management/http/HttpCassandraManagementProxy.java

```diff
@@ -405,7 +408,9 @@ public List<String> getLiveNodes() throws ReaperException {
       List<String> liveNodes = new ArrayList<>();
       EndpointStates endpoints = apiClient.getEndpointStates();
       for (Map<String, String> states : endpoints.getEntity()) {
-        if (states.containsKey("IS_ALIVE") && Boolean.parseBoolean(states.get("IS_ALIVE"))) {
+        if (states.containsKey("IS_ALIVE")
+            && Boolean.parseBoolean(states.get("IS_ALIVE"))
+            && !isCoordinatorNode(states)) {
```

Comment:
suggestion:
I think it would be a small optimization to check if it's a coordinator node first as the states should almost always have an "IS_ALIVE" key. For coordinator-only nodes(i.e. Stargate), changing the order of the checks here will reduce the number of map reads I think.
```suggestion
        if (!isCoordinatorNode(states)
            && states.containsKey("IS_ALIVE")
            && Boolean.parseBoolean(states.get("IS_ALIVE"))) {
```

Reply:
addressed 👍